subgraph-js/src/schema-helper/buildSchemaFromSDL.ts

```diff
@@ -196,6 +196,7 @@ export function buildSchemaFromSDL(
   const schemaDefinitions: SchemaDefinitionNode[] = [];
   const schemaExtensions: SchemaExtensionNode[] = [];
   const schemaDirectives: ConstDirectiveNode[] = [];
+  const schemaDescriptions: string[] = []
```

Comment:
Can't there only be one schema description? This should probably just be a `string | undefined`.

Reply:
Yes, I think you're effectively correct in practice. The only confusion is that `buildSchemaFromSDL` actually allows you to pass `DocumentNode | DocumentNode[]`, and so in the latter case there's presumably the chance that we get more than one description?

It seems odd that it works that way though, and I notice that you've recently imported the `buildSchemaFromSDL` function from a stand-along module, and so maybe this array form is no longer needed in its new setting, in which case I'm definitely happy to get rid of as it really does seem wrong as you say?

Reply:
In the `DocumentNode[]` case, you can have one `schema` definition as well as `extend schema` definition extensions. The `extend schema` notation is not permitted to have a description attached.

https://spec.graphql.org/draft/#sec-Schema
>When using the type system definition language, a document must include at most one schema definition.

https://spec.graphql.org/draft/#sec-Schema-Extension
Note that there's no description permitted on the grammar for extensions